config/locales/en.yml

```diff
@@ -8,3 +8,15 @@ en:
       processing: "Processing"
       succeeded: "Succeeded"
       failed: "Failed"
+    shared:
```

Comment:
Since it is shared, it could moved at the root level:

```
en:
  keywords:
    ...
  shared:
    ...
```

Reply:
Fixed in b58d270738575daaea4ef2a0682c10ebc33b0f31